data/autoyast_sle15/pc_tools.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>!!!!!!</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+        <addons config:type="list">
+            <addon>
+                <name>sle-module-basesystem</name>
+                <version>15</version>
```

Comment:
This settings can be also injected dynamically  in case you plan to run it for more versions, e.g. 15.1

Reply:
no this would be static setup . we don't plan to change versions in nearest future . this image is not System Under Test . it is "platform" used to reach SUT which is inside Public Cloud